src/model/implementation/field.ts

```diff
@@ -1687,6 +1687,20 @@ export class Field implements ModelComponent {
                 ),
             );
         }
+        if (
+            this.isIncludedInSearch &&
+            this.type.isObjectType &&
+            !this.type.fields.some(
+                (value) => value.isIncludedInSearch,
+            )
+        ) {
+            context.addMessage(
+                ValidationMessage.warn(
+                    `At least one field on type "${this.type.name}" should be marked with "includeInSearch".`,
+                    this.input.isFlexSearchIndexedASTNode ?? this.input.isFlexSearchFulltextIndexedASTNode,
```

Comment:
suggestion for the message

    "includeInSearch: true" does not have an effect because none of the fields in type "${this.type.name}" have "includeInSearch: true".

would be nice to report this on the "includeInSearch" argument, but with the message referencing the argument, that might also be enough to understand it.